mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -271,7 +273,7 @@ object OneVsRestModel extends MLReadable[OneVsRestModel] {
 @Since("1.4.0")
 final class OneVsRest @Since("1.4.0") (
     @Since("1.4.0") override val uid: String)
-  extends Estimator[OneVsRestModel] with OneVsRestParams with MLWritable {
+  extends Estimator[OneVsRestModel] with OneVsRestParams with HasParallelism with MLWritable {
 
```

Comment:
@WeichenXu123 earlier in the discussion for this PR we had been coordinating with similar changes for parallel cross validator and train validation split at #16774 .

If `trait OneVsRestParams extends PredictorParams with HasParallelism` was done instead, then it would just give the parallelism param to the `OneVsRestModel` which isn't really useful since it's already been trained.